common.go

```diff
@@ -22,14 +23,14 @@ import (
 )
 
 const (
-	VersionSSL30        = 0x0300
-	VersionTLS10        = 0x0301
-	VersionTLS11        = 0x0302
-	VersionTLS12        = 0x0303
-	VersionTLS13        = 0x0304
-	VersionTLS13Draft18 = 0x7f00 | 18
-	VersionTLS13Draft21 = 0x7f00 | 21
-	VersionTLS13Draft22 = 0x7f00 | 22
+	VersionSSL30        = constants.VersionSSL30
```

Comment:
The correct number of definitions for "VersionSSL30" variable is 1 or less :) 
("or less" being prefered:)